src/main/java/cinema/menu/Menu.java

```diff
@@ -57,43 +61,27 @@ private void showFilms() {
     }
 
     private void registration() {
-        System.out.println("введите логин");
+        System.out.println(ASK_FOR_LOGIN);
         String login = reader.nextLine();
-        System.out.println("введите пароль");
+        System.out.println(ASK_FOR_PASSWORD);
         String password = reader.nextLine();
         User user = new User(login, password);
         userDao.save(user);
     }
 
     private void authentication() {
-        System.out.println("введите логин");
+        System.out.println(ASK_FOR_LOGIN);
         String login = reader.nextLine();
-        System.out.println("введите пароль");
+        System.out.println(ASK_FOR_PASSWORD);
         String password = reader.nextLine();
-        String sql_id_by_login_and_password = "select id from users where login='" + login + "' and password='" + password + "'";
-        Long userId =0L;
-        try (Connection connection = ConnectionManager.get()) {
-            Statement statement = connection.createStatement();
-            ResultSet resultSet = statement.executeQuery(sql_id_by_login_and_password);
-            while (resultSet.next()) {
-                userId = resultSet.getLong("id");
-            }
-            Optional<User> userDaoById = userDao.findById(userId);
-
-            if (userDaoById.isPresent())
-                userActionMenu.userMenu(userDaoById);
-            else throw new RuntimeException();
-        } catch (SQLException e) {
-            e.printStackTrace();
-        }
+        userActionMenu.userMenu(userDao.userAuthenticationByLoginAndPassword(login, password));
     }
 
-
     private void mainMenu() {
-        System.out.println("===МЕНЮ===" + "\n" +
-                "1. войти" + "\n" +
-                "2. регистрация" + "\n" +
-                "3. просмотреть мероприятия(фильмы)" + "\n" +
-                "4. выход");
+        System.out.println("===MENU===" + "\n" +
```

Comment:
"===MENU===" + "\n" +
                "1. enter" + "\n" +
                "2. registration" + "\n" +
                "3. view events(movies)" + "\n" +
                "4. quit" - may be  is this  variable too?